runners/samza/src/main/java/org/apache/beam/runners/samza/runtime/OpAdapter.java

```diff
@@ -102,7 +115,16 @@ public synchronized CompletionStage<Collection<OpMessage<OutT>>> apply(OpMessage
               String.format("Unexpected input type: %s", message.getType()));
       }
     } catch (Exception e) {
-      LOG.error("Op {} threw an exception during processing", this.getClass().getName(), e);
+      LOG.error("Op {} threw an exception during processing", transformFullName, e);
+      e.addSuppressed(
+          new RuntimeException(
+              String.format("Op: %s threw an exception during processing", transformFullName)));
```

Comment:
Wording: "Exception happened in transform %s", ...

Reply:
done